Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -2292,6 +2294,19 @@ RemoteMediaEngineConfigurationFactory& WebProcess::mediaEngineConfigurationFacto
 }
 #endif
 
+std::optional<IPC::Connection::UniqueID> WebProcess::networkProcessConnectionID()
+{
+    Locker lock { m_lockNetworkProcessConnectionID };
+    return m_networkProcessConnectionID;
+}
+
+void WebProcess::setNetworkProcessConnectionID(std::optional<IPC::Connection::UniqueID> uniqueID)
```

Comment:
I don't think this should be using a std::optional either.

Reply:
Resolved in latest patch.